### PR TITLE
LogContext: Improve text describing the loglines

### LINKED ISSUE #55472: Update the copy which previously said 'Found 10 rows'
Description in https://github.com/grafana/grafana/issues/50270.

### PR DESCRIPTION
**Pull request by svennergr**
**What this PR does / why we need it**:
Described in https://github.com/grafana/grafana/issues/50270. Change the text `Found 10 rows` to `Showing n lines before/after match`.

**Which issue(s) this PR fixes**:

Fixes #55472

### CODE REVIEW

File: public/app/features/logs/components/LogRowContext.test.tsx [hunk 1]
```diff
@@ -22,4 +23,29 @@ describe('LogRowContextGroup component', () => {
...
+      onLoadMoreContext: () => {},
+      canLoadMoreRows: false,
+      row: {} as LogRowModel,
+      className: '',
+      groupPosition,
+      logsSortOrder,
+    };
+
+    render(
+      <div>
```
Comment (matyax) [inline public/app/features/logs/components/LogRowContext.test.tsx:44]: Do you really need this div?

File: public/app/features/logs/components/LogRowContext.tsx [hunk 2]
```diff
@@ -74,31 +80,54 @@ interface LogRowContextGroupHeaderProps {
...
   onLoadMoreContext,
   canLoadMoreRows,
+  groupPosition,
+  logsSortOrder,
 }) => {
   const { header } = useStyles2(getLogRowContextStyles);
 
+  // determine the position in time for this LogGroup by taking the ordering of
+  // logs and position of the component itself into account.
+  let logGroupPosition = '';
```
Comment (matyax) [inline public/app/features/logs/components/LogRowContext.tsx:108]: Not really in love with this block.

What about:

```javascript
let logGroupPosition = 'after';
if (groupPosition === LogGroupPosition.Bottom) {
  if (logsSortOrder === LogsSortOrder.Descending) {
    logGroupPosition = 'before';
  }
} else if (logsSortOrder === LogsSortOrder.Ascending) {
  logGroupPosition = 'before';
}
```

**Review by matyax — COMMENTED** [review]
Looks good! Left a suggestion.

File: public/app/features/logs/components/LogRowContext.tsx (hunk 2 above)
Comment (svennergr) [reply public/app/features/logs/components/LogRowContext.tsx]: I was looking at those (my) `if`s for 10minutes and did not came up with that. Thanks a lot, that looks way better! 

Fixed in https://github.com/grafana/grafana/pull/55475/commits/6018cbe4ef9c7e2882f89ad89e1b420bad935a64.

**Review by matyax — APPROVED** [review]
👌

**Pull request merged**